doc/node.1

```diff
@@ -1,36 +1,30 @@
 .TH NODE 1 2016 Node.js Node.js
-
 .\" This is a man page comment.
-
+.\"
 .\" Man page syntax (actually roff syntax) is somewhat obscure, but the
 .\" important part is is that .<letter> specifies <letter>'s syntax for that
 .\" line, and \f<letter> specifies it for the characters that follow.
-
+.\"
 .\" .B   Bold line
 .\" .I   Italic line (Rendered as underlined text in terminals)
 .\" .BI  Alternating bold/italics without spaces between arguments.
 .\"      Use `\ ` to include an "unpaddable" (literal) space in the output.
 .\" .RI  Alternating roman/italic
-
+.\"
 .\" See http://liw.fi/manpages/ for an overview, or http://www.troff.org/54.pdf
 .\" for detailed language reference.
-
+.\"
 .\" Macro to display an underlined URL in bold
 .de ur
 .nr CF \\n(.f
 .ft 4
 \\$1
 .ft \\n(CF
 ..
-
-
```

Comment:
Sorry, I meant not to touch these at all.

I _know_ it's not "perfectly syntactically correct" but it is far more readable.

In my experience the problems with rendering are negligible if any for the potential cost of people having a hard time contributing.